js/main.js

```diff
@@ -1,330 +0,0 @@
-// main.js
-console.log('main.js')
-
-// global card counter
-let cardNum = 0;
-
-//global array, keep track of zip codes
-zipArray = [];
-
-// Set const to body and assign id with "bodyid"
-let htmlBody = document.querySelector('body');
-htmlBody.setAttribute('id','bodyid');
-
-// Create const mainDiv holds all html elements for weather app
-const mainDiv = document.createElement('div');
-mainDiv.setAttribute('id','main');
-htmlBody.appendChild(mainDiv);
-
-// Create a set of headerDiv and bodyDiv for our content
-// Use as reference for when appending childs
-const headerDiv = document.createElement('div');
-headerDiv.setAttribute('id', 'headerDiv');
-htmlBody.appendChild(headerDiv);
-const bodyDiv = document.createElement('div');
-bodyDiv.setAttribute('id', 'bodyDiv');
-htmlBody.appendChild(bodyDiv);
-
-
-// testing append on body
-// htmlBody.appendChild(mainDiv)
-// htmlBody.appendChild(titleTextElem)
-
-// Function for creating html elements for DOM with JS
-// Create a func with Params passing
-// html element name and name of the id
-// then append element to a parent Param passed in the function
-function createElem(htmlElem, idName, parentElem, className = null){
-    const parent = document.getElementById(parentElem)
-    const makeElem = document.createElement(htmlElem);
-    makeElem.setAttribute('id', idName); 
-    if (className !== null)
-    makeElem.setAttribute('class', className)
-    parent.appendChild(makeElem);
-
-    //return parentElem;
```

Comment:
This would be a good idea to be able to return the element and set it to a variable. Which could cut down on the params you pass to this function by...
```javascript
let parent1 = createElem(...args)
let child1 = createElement(parent1, ...args)